2009-11-22  Michael Tautschnig  <mt@debian.org>

	* install_packages: Added zypper support (thanks Thomas Neumann for the
		patch). (closes: #514160)
Index: trunk/bin/install_packages
===================================================================
--- trunk.orig/bin/install_packages
+++ trunk/bin/install_packages	
@@ -69,7 +69,7 @@
 
 
 # @commands is the order of the commands that are executed
-our @commands = qw/y2i y2r yast rpmr urpmi urpme yumgroup yumi yumr smarti smartr hold taskrm taskinst clean-internal aptitude aptitude-r install unpack remove dselect-upgrade/;
+our @commands = qw/y2i y2r zypper yast rpmr urpmi urpme yumgroup yumi yumr smarti smartr hold taskrm taskinst clean-internal aptitude aptitude-r install unpack remove dselect-upgrade/;
 %command = (
           "install" => "apt-get $aptopt --fix-missing install",
     "inst-internal" => "apt-get $aptopt --fix-missing -s install",
@@ -90,6 +90,7 @@
 	     "yumi" => "yum -y install",
 	     "yumr" => "yum -y remove",
         "yumgroup"  => "yum -y groupinstall",
+	  "zypper"  => "zypper -n install",
 	     "y2i"  => "y2pmsh isc",
 	     "y2r"  => "y2pmsh remove",
 	     "yast" => "yast -i",
